system-parachains/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -244,3 +253,120 @@ impl pallet_salary::Config<FellowshipSalaryInstance> for Runtime {
 	// Total monthly salary budget.
 	type Budget = ConstU128<{ 100_000 * USDT_UNITS }>;
 }
+
+parameter_types! {
+	// TODO: reference the constant value from common crate with polkadot-sdk 1.5
+	pub const FellowshipTreasuryPalletId: PalletId = PalletId(*b"py/feltr");
+	pub const ProposalBond: Permill = Permill::from_percent(1);
+	pub const ProposalBondMinimum: Balance = 5 * DOLLARS;
+	pub const ProposalBondMaximum: Balance = 10 * DOLLARS;
```

Comment:
I think these can be lower than the [Relay Chain deposits](https://github.com/polkadot-fellows/runtimes/blob/16e556492fd0d8c9dd54cdd9a2ec4a2d7fb8aeb0/relay/polkadot/src/lib.rs#L710), but still similar order of magnitude. Just a suggestion:
```suggestion
	pub const ProposalBondMinimum: Balance = 20 * DOLLARS;
	pub const ProposalBondMaximum: Balance = 200 * DOLLARS;
```

Reply:
updated